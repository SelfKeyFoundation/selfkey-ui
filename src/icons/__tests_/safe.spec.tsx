import * as React from 'react';
import { create } from 'react-test-renderer';
import { SafeIcon } from '../safe';

describe('Safe Icon', () => {
	it('should show Safe Icon', () => {
		const component = create(<SafeIcon />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
