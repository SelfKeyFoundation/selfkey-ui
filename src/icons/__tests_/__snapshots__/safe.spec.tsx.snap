// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Safe Icon should show Safe Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root Component-root-1"
  focusable="false"
  style={
    Object {
      "height": "55px",
      "width": "66px",
    }
  }
  viewBox="0 0 66 55"
>
  <title>
    5BAAB12E-9ECD-4338-9F8A-DE8E1925DCCB
  </title>
  <defs>
    <linearGradient
      id="linearGradient-1"
      x1="11.0842631%"
      x2="36.0705554%"
      y1="62.7357153%"
      y2="15.2777778%"
    >
      <stop
        offset="0%"
        stopColor="#09A8BA"
      />
      <stop
        offset="100%"
        stopColor="#0ABBD0"
      />
    </linearGradient>
  </defs>
  <g
    fill="none"
    fillRule="evenodd"
    id="ðŸ› -Wallet-Setup-Passphrase"
    stroke="none"
    strokeWidth="1"
  >
    <g
      fill="url(#linearGradient-1)"
      fillRule="nonzero"
      id="0108-Backup-Wallet"
      transform="translate(-375.000000, -387.000000)"
    >
      <g
        id="Modal"
        transform="translate(330.000000, 291.000000)"
      >
        <path
          d="M48.3,96 C46.43,96 45,97.43 45,99.3 L45,143.3 C45,145.17 46.43,146.6 48.3,146.6 L53.8,146.6 L53.8,148.8 C53.8,150.01 54.79,151 56,151 L60.4,151 C61.61,151 62.6,150.01 62.6,148.8 L62.6,146.6 L93.4,146.6 L93.4,148.8 C93.4,150.01 94.39,151 95.6,151 L100,151 C101.21,151 102.2,150.01 102.2,148.8 L102.2,146.6 L107.7,146.6 C109.57,146.6 111,145.17 111,143.3 L111,99.3 C111,97.43 109.57,96 107.7,96 L48.3,96 Z M48.3,98.2 L107.7,98.2 C108.36,98.2 108.8,98.64 108.8,99.3 L108.8,143.3 C108.8,143.96 108.36,144.4 107.7,144.4 L100,144.4 L95.6,144.4 L60.4,144.4 L56,144.4 L48.3,144.4 C47.64,144.4 47.2,143.96 47.2,143.3 L47.2,99.3 C47.2,98.64 47.64,98.2 48.3,98.2 Z M54.9,102.6 C53.03,102.6 51.6,104.03 51.6,105.9 L51.6,136.7 C51.6,138.57 53.03,140 54.9,140 L101.1,140 C102.97,140 104.4,138.57 104.4,136.7 L104.4,135.6 C104.4,134.94 103.96,134.5 103.3,134.5 C102.64,134.5 102.2,134.94 102.2,135.6 L102.2,136.7 C102.2,137.36 101.76,137.8 101.1,137.8 L54.9,137.8 C54.24,137.8 53.8,137.36 53.8,136.7 L53.8,105.9 C53.8,105.24 54.24,104.8 54.9,104.8 L101.1,104.8 C101.76,104.8 102.2,105.24 102.2,105.9 L102.2,107 C102.2,107.66 102.64,108.1 103.3,108.1 C103.96,108.1 104.4,107.66 104.4,107 L104.4,105.9 C104.4,104.03 102.97,102.6 101.1,102.6 L54.9,102.6 Z M102.2,109.2 C100.99,109.2 100,110.19 100,111.4 L100,115.8 C100,117.01 100.99,118 102.2,118 L104.4,118 C105.61,118 106.6,117.01 106.6,115.8 L106.6,111.4 C106.6,110.19 105.61,109.2 104.4,109.2 L102.2,109.2 Z M78,110.3 C71.95,110.3 67,115.25 67,121.3 C67,127.35 71.95,132.3 78,132.3 C84.05,132.3 89,127.35 89,121.3 C89,115.25 84.05,110.3 78,110.3 Z M57.1,111.4 C56.44,111.4 56,111.84 56,112.5 L56,130.1 C56,130.76 56.44,131.2 57.1,131.2 C57.76,131.2 58.2,130.76 58.2,130.1 L58.2,112.5 C58.2,111.84 57.76,111.4 57.1,111.4 Z M102.2,111.4 L104.4,111.4 L104.4,115.8 L102.2,115.8 L102.2,111.4 Z M78,112.280859 C82.95,112.280859 87.0191409,116.35 87.0191409,121.3 C87.0191409,126.25 82.95,130.319141 78,130.319141 C73.05,130.319141 68.9808591,126.25 68.9808591,121.3 C68.9808591,116.35 73.05,112.280859 78,112.280859 Z M78,113.6 C77.34,113.6 76.9,114.04 76.9,114.7 L76.9,116.680859 C76.9,117.340859 77.34,117.780859 78,117.780859 C78.66,117.780859 79.1,117.340859 79.1,116.680859 L79.1,114.7 C79.1,114.04 78.66,113.6 78,113.6 Z M73.3808591,115.469141 C73.1058591,115.469141 72.8295699,115.58 72.6095699,115.8 C72.1695699,116.24 72.1695699,116.90043 72.6095699,117.34043 L74.0404301,118.769141 C74.2604301,118.989141 74.5895699,119.1 74.8095699,119.1 C75.0295699,119.1 75.3608591,118.989141 75.5808591,118.769141 C76.0208591,118.329141 76.0208591,117.670859 75.5808591,117.230859 L74.15,115.8 C73.93,115.58 73.6558591,115.469141 73.3808591,115.469141 Z M82.7308591,115.469141 C82.4558591,115.469141 82.1795699,115.58 81.9595699,115.8 L80.5308591,117.230859 C80.0908591,117.670859 80.0908591,118.329141 80.5308591,118.769141 C80.7508591,118.989141 80.97,119.1 81.3,119.1 C81.63,119.1 81.8491409,118.989141 82.0691409,118.769141 L83.5,117.34043 C83.94,116.90043 83.94,116.24 83.5,115.8 C83.28,115.58 83.0058591,115.469141 82.7308591,115.469141 Z M103.3,119.1 C102.64,119.1 102.2,119.54 102.2,120.2 L102.2,122.4 C102.2,123.06 102.64,123.5 103.3,123.5 C103.96,123.5 104.4,123.06 104.4,122.4 L104.4,120.2 C104.4,119.54 103.96,119.1 103.3,119.1 Z M71.4,120.2 C70.74,120.2 70.3,120.64 70.3,121.3 C70.3,121.96 70.74,122.4 71.4,122.4 L73.3808591,122.4 C74.0408591,122.4 74.4808591,121.96 74.4808591,121.3 C74.4808591,120.64 74.0408591,120.2 73.3808591,120.2 L71.4,120.2 Z M82.6191409,120.2 C81.9591409,120.2 81.5191409,120.64 81.5191409,121.3 C81.5191409,121.96 81.9591409,122.4 82.6191409,122.4 L84.6,122.4 C85.26,122.4 85.7,121.96 85.7,121.3 C85.7,120.64 85.26,120.2 84.6,120.2 L82.6191409,120.2 Z M81.1904301,123.280859 C80.9154301,123.280859 80.6391409,123.38957 80.4191409,123.60957 C79.9791409,124.04957 79.9791409,124.71 80.4191409,125.15 L81.85,126.580859 C82.07,126.800859 82.3991409,126.90957 82.6191409,126.90957 C82.8391409,126.90957 83.1704301,126.800859 83.3904301,126.580859 C83.8304301,126.140859 83.8304301,125.48043 83.3904301,125.04043 L81.9595699,123.60957 C81.7395699,123.38957 81.4654301,123.280859 81.1904301,123.280859 Z M74.8095699,123.39043 C74.5345699,123.39043 74.2604301,123.499141 74.0404301,123.719141 L72.6095699,125.15 C72.1695699,125.59 72.1695699,126.25043 72.6095699,126.69043 C72.8295699,126.91043 73.1608591,127.019141 73.3808591,127.019141 C73.6008591,127.019141 73.93,126.91043 74.15,126.69043 L75.5808591,125.25957 C76.0208591,124.81957 76.0208591,124.159141 75.5808591,123.719141 C75.3608591,123.499141 75.0845699,123.39043 74.8095699,123.39043 Z M78,124.6 C77.34,124.6 76.9,125.04 76.9,125.7 L76.9,127.680859 C76.9,128.340859 77.34,128.780859 78,128.780859 C78.66,128.780859 79.1,128.340859 79.1,127.680859 L79.1,125.7 C79.1,125.04 78.66,124.6 78,124.6 Z M102.2,124.6 C100.99,124.6 100,125.59 100,126.8 L100,131.2 C100,132.41 100.99,133.4 102.2,133.4 L104.4,133.4 C105.61,133.4 106.6,132.41 106.6,131.2 L106.6,126.8 C106.6,125.59 105.61,124.6 104.4,124.6 L102.2,124.6 Z M102.2,126.8 L104.4,126.8 L104.4,131.2 L102.2,131.2 L102.2,126.8 Z M56,146.6 L60.4,146.6 L60.4,148.8 L56,148.8 L56,146.6 Z M95.6,146.6 L100,146.6 L100,148.8 L95.6,148.8 L95.6,146.6 Z"
          id="icon-safe"
        />
      </g>
    </g>
  </g>
</svg>
`;
